### PR TITLE
treehouses led carneval (fixes #593)

### LINKED ISSUE #593: treehouses led carneval?

### PR DESCRIPTION
**Pull request by jamjakpa**
please test the led.sh
I have added the Carneval LED blink , please test.
Function called 'carneval'

### CODE REVIEW

File: modules/led.sh [hunk 1]
```diff
@@ -38,6 +38,12 @@ function led {
     echo "Look at your RPi leds, both leds will be in this pattern... "
     echo "Both LED: 1 sec on; 8 blink; 1 on"
     christmas > /dev/null
+  elif [ "$color" = "carneval" ]; then
+    checkroot
+    echo "leds are set to carnval mode."
+    echo "Look at your RPi leds, both leds will be in this pattern... "
+    echo "Both LED: 1 sec on; 8 blink; 1 on"
```
Comment (kjong) [inline modules/led.sh]: You should change the timing descriptions to match yours.

Comment (kjong) [reply modules/led.sh]: Also, `carneval` is spelled `carnival` unless you're referring to something else.

Comment (jamjakpa) [reply modules/led.sh]: Hi kjong, I corrected the spell ( carnival ) . thanks for letting me know.

Comment (jamjakpa) [reply modules/led.sh]: I also change the timing descriptions

**Pull request merged**